### DISCUSSION TITLE
Info about maps, navigation, etc. by a former Kurviger user

### DISCUSSION THREAD

**Discussion by ramondevic** [devemux86/cruiser#33] [Q&A] [answered]
Hi, I'm new here and interested in the new Cruiser app.
I've been using the Kurviger apps for a long time now and I'm very happy with them and I plan to continue with the Cruiser, in fact I already have it installed on my phone but haven't tried it much yet.

Used to Kurviger I find big differences in terms of maps: before I downloaded the maps from https://download.kurviger.de/maps/ using the browser and now from the app I see that they can be downloaded from Mapsforge, OpenAndroMaps and Freitzkarte .
Hey, I saw that you can use the same maps from Mapsforge that you had already downloaded and you can configure the path where they are, great!

As for routes, if I had access to internet data, the app would already calculate the route for me according to the option I could choose: faster, faster and curves, curves, many curves and all curves.
If I didn't have Interner, I would have previously downloaded the Brouter data and calculated the route, usually the fastest route.
Normally if I prepare a route from the pc I do it from the Kurviger website and then export it as ".kurviger" and open it from the app, that's great!

But when I'm on the road and I want to go somewhere I don't know in Cruiser how to select the type of route I want to take (fast, many curves, etc).

Surely there is some manual explaining how all this works: maps, routes, tips for those of us who come from Kurviger and feel very comfortable with the old app.
What are Graphs?
If there is no Internet in the App, are the same Brouter files used?

Sorry for the rambling, but I'm looking forward to going on a trip to Asia at the beginning of May, I'm preparing the routes in Kurviger to export to Cruiser and I want to be as prepared and tested as possible by the time I go.

(Sorry, translated from Catalan to English with Google)

**Comment by devemux86** [comment, marked as answer]
Hi @ramondevic!

---

Cruiser has been on Google Play since 2013.

Kurviger 2+1 used the _Cruiser libraries_.
Kurviger 3 uses Mapbox for navigation.

---

Cruiser has offline maps with integrated downloader from Mapsforge | [OpenAndroMaps](https://www.openandromaps.org/en/) | [Freizeitkarte](https://www.freizeitkarte-osm.de/android/en/index.html) + their themes.

Mapsforge maps have been updated and now contain the same map layers and zoom levels as you knew.
You can still use external map files from anywhere (enable the "Settings | Application | Menu | Advanced").

Cruiser also has the familiar "**Biker**" theme with the **motorcycle** map layers.

Cruiser can **load offline kurviger routes** (not Kurviger routing).

In navigation you can use **offline re-routing** with BRouter or GraphHopper (use integrated downloader to get its routing graphs).
Cruiser re-routing only calculates the path from the user location to the next waypoint, the rest of the **route remains unchanged**.

---

_Development on Cruiser continues, new features are added all the time._

Like the **navigation profiles** where you can select your vehicle profile, default speed,
customize voice instructions and announcement time (for details see discussion https://github.com/devemux86/cruiser/discussions/19).

**Route overview** in navigation (tap navigation panel) and many other new features:
**Skip many waypoints** (see https://github.com/devemux86/cruiser/discussions/17), **Route duration from your profile** (see https://github.com/devemux86/cruiser/discussions/28), etc.

I hope to answered your questions!
Please ask again anything you want to know.

**Comment by DolomitesTom** [comment]
> Cruiser can load offline kurviger routes (cannot use the Kurviger routing).

The workaround is to place more shaping points (not really a problem). But Cruiser has already more fancy stuff beyond K2. Next version has already some nice features like layout offset for buttons and navigation bars. Later will get little statistics like average speed and tour duration calculated on average speed and so on.

**Comment by ramondevic** [comment]
I have some doubts due to the different options that were not in the K2:

Routes menu settings, there are the following options:
<img src="https://user-images.githubusercontent.com/122691401/218143553-f692888d-4c58-41ce-9248-ef430a295433.png" width="50%"/>

I would like to know the difference between choosing one or the other without having to choose one by one with the same destination to try to discover the differences.
Options to Search:

The same with: 
<img src="https://user-images.githubusercontent.com/122691401/218143666-c344cae1-337e-486d-b9de-90a7fce3045e.png" width="50%"/><img src="https://user-images.githubusercontent.com/122691401/218143725-6d1fddb4-71ae-4935-b236-ed42197de0ad.png" width="50%"/>

What is the predefined speed for ?
Car, Motorbike, Bycicle, ...

<img src="https://user-images.githubusercontent.com/122691401/218143948-b37c7fe0-1dde-48f9-b462-23a9a79c6154.png" width="50%"/>

In Routes menu download Graphs ?
What is this for ? For calculating route offline ?

<img src="https://user-images.githubusercontent.com/122691401/218145405-09ab0d2c-a594-42ff-9472-02bdd63a01a2.png" width="50%"/>

Thanks !

**Reply by devemux86** [reply]
@ramondevic
_I edited your post to make the images smaller for easier reading._
(see https://github.com/devemux86/cruiser/discussions/27)

**Comment by devemux86** [comment]
> I would like to know the difference between choosing one or the other without having to choose one by one with the same destination to try to discover the differences.

All routing services can calculate routes for: Car / Motorcycle, Bike, Foot.
[CycleStreets](https://www.cyclestreets.net/) is specifically for bikes (quietest / balanced routes).
OpenRouteService and Valhalla offer some avoid options.

The most flexible routing is BRouter (offline) with its customizable profiles.
All routing services have websites to easier test them.

> The same with:

Search has separate options, can try them or better stay with the default.
In "API keys" can enter your own private key if register with their website.

> What is the predefined speed for ?

Please see the announcement in https://github.com/devemux86/cruiser/discussions/19, also https://github.com/devemux86/cruiser/discussions/19#discussioncomment-4754379 and https://github.com/devemux86/cruiser/discussions/19#discussioncomment-4756123.

If you need more information about it, please tell us.

> In Routes menu download Graphs ?
> What is this for ? For calculating route offline ?

Maybe we should name it differently. :slightly_smiling_face: 
It is for downloading offline routing data for GraphHopper (offline) routing.

Cruiser supports 2 offline routing engines: BRouter and GraphHopper:
- BRouter data are downloaded with BRouter app
- GraphHopper data are downloaded in Cruiser

---

If you are interested mainly for navigation with offline re-routing:
- You can load (offline) your kurviger routes
- And select BRouter (offline) for re-routing

All other options can remain the defaults.

Navigation profiles and default speeds currently affect the routing and voice navigation.
They will play a bigger role in the future, see the new route duration from profiles in https://github.com/devemux86/cruiser/discussions/28.

**Reply by ramondevic** [reply]
This weekend I've been testing Cruiser with online routing with the different routing options and I'm happy with it.
In fact, when I prepare a trip or a long outing, I prepare it from the Kurviger website and then download the file to browse it offline ... and I will continue to do so, I find it to be the most agile and functional.

If when I want to go to a specific place (from A to B) with Cruiser it works correctly and if you don't like the path that Brouter chooses (or one of the others) you can always put an intermediate point along the road that interests you.
For me, this way of doing things is what I find ideal.

... just one problem ... see if you can change the app icon ;)

**Reply by devemux86** [reply]
@ramondevic
Very nice you like it!

If you have suggestions for improvements or new features, please write them.
I always collect and analyze all reports for future implementation. :slightly_smiling_face:

**Comment by ToToGSARider** [comment]
@devemux86 hey ho, welche Einstellungen im Cruiser muß ich setzen, damit die Route von der Kurviger Internetseite genau wie in der K1 und K2 ohne Änderungen übernommen werden. Die Einstellung Neuberechnung aus habe ich schon gesetzt. 

@devemux86  hey ho, what settings in the Cruiser do I have to set so that the route from the Kurviger website are taken over exactly as in the K1 and K2 without changes. I have already set the recalculation off setting.

**Reply by devemux86** [reply]
@ToToGSARider 

You can use the menu "Routing | Import" to import offline your Kurviger routes.

Cruiser re-routing only calculates the path from the user location to the next waypoint, the rest of the route remains unchanged.

You can download offline maps using the integrated downloader and use the "Biker" map theme and layers.

Please also see the images and ask again anything you want to know!

---

@ToToGSARider 

Sie können das Menü "Routing | Importieren" verwenden, um Ihre Kurviger-Routen offline zu importieren.

Cruiser Re-Routing berechnet nur den Weg vom Benutzerstandort zum nächsten Wegpunkt, der Rest der Route bleibt unverändert.

Sie können Offline-Karten mit dem Karten-Downloader herunterladen und das Kartenthema "Biker" sowie die Ebenen verwenden.

Bitte sehen Sie sich auch die Bilder an und fragen Sie noch einmal alles, was Sie wissen wollen!

<img src="https://user-images.githubusercontent.com/3484020/227294020-8cd23ee0-beb6-4d9e-a921-63f98184e044.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/227294050-0d945913-0469-4029-88d4-21c898ebb4f8.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/227294072-9b8e4c8a-6623-4cf6-8715-c0eabfd94f37.jpg" width="33%"/>

**Reply by ToToGSARider** [reply]
Hallo,

Bei mir klappt der Import der .kurviger Dateien nicht aus der App heraus wenn ich die Import Funktion nutze. Wenn ich aber in den Speicherplatz auf dem Handy gehe und dort die Cruiser App auswähle, klappt der Import direkt. Ist das ein Bug oder mache ich was falsch. Ich exportiere die Route ganz normal als .kurviger Datei von der Internetseite. 

Hello,

For me, the import of .kurviger files does not work from the app when I use the import function. But if I go into the storage space on the phone and select the Cruiser app there, the import works directly. Is this a bug or am I doing something wrong. I export the route normally as a .curv file from the website.

**Reply by devemux86** [reply]
Do you use the menu "Routing | Import", select the .kurviger file and see the 2nd image?
Can you attach a .kurviger file that does not work?

---

Verwenden Sie das Menü "Routing | Importieren", wählen Sie die .kurviger Datei aus und sehen Sie das zweite Bild?
Können Sie eine .kurviger Datei anhängen, die nicht funktioniert?

<img src="https://user-images.githubusercontent.com/3484020/227294072-9b8e4c8a-6623-4cf6-8715-c0eabfd94f37.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/227314153-e8259162-1f30-4bdf-b49c-b198e98a12ed.jpg" width="33%"/>

**Reply by ToToGSARider** [reply]
Kann es daran liegen, dass ich noch kein ABO abgeschlossen habe? Bei mir wird nur angezeigt Routing / Import löschen / und wenn ich auf Import drücke geht direkt die Auswahl Speicherplatz auf dem Handy auf.

Can it be because I have not yet subscribed? For me it is only displayed Routing / Import delete / and when I press Import goes directly to the selection storage space on the phone.

**Reply by devemux86** [reply]
The subscription is for the Navigation + GPS recording.

It is the same process with Kurviger apps:

- Press import
- Android file browser opens
- Select the kurviger file to import

Tthe other menu appear when there is a route.

---

Das Abonnement gilt für die Navigation + GPS-Aufzeichnung.

Bei Kurviger-Apps ist es derselbe Prozess:

- Drücken Sie importieren
- Android Dateibrowser öffnet sich
- Wählen Sie die zu importierende Kurviger-Datei aus

Das andere Menü erscheint, wenn es eine Route gibt.

**Reply by 1maxi2** [reply]
<img src="https://user-images.githubusercontent.com/124886589/227334308-7ab4baba-fbe6-4bf5-9927-8359bccf3f7d.jpg" width="33%"/>
Wie komme ich zu dieser Auswahlmöglichkeit in Cruiser wie im Screenshot zu sehen ?
Lg

**Reply by devemux86** [reply]
@1maxi2 

Do you mean how to open the menu?
Press the menu button at bottom left.

---

Sie meinen, wie man das Menü öffnet?
Drücken Sie die Menütaste unten links.

**Reply by 1maxi2** [reply]
<img src="https://user-images.githubusercontent.com/124886589/227338153-07a5df09-1516-4770-8b2c-baaf02aeefd8.jpg" width="33%"/>

??
Was mache ich falsch ?

**Reply by devemux86** [reply]
When there is not loaded an offline map or route, the menu appear like this:

---

Wenn keine Offline-Karte oder -Route geladen ist, sieht das Menü wie folgt aus:

<img src="https://user-images.githubusercontent.com/3484020/227338057-42382a20-1ffc-4b83-a9b7-4c4f67b657f9.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/227338102-3eaddc18-5c6d-45c1-b9a7-a94f39081700.jpg" width="33%"/>

**Reply by devemux86** [reply]
@1maxi2 

> Was mache ich falsch ?

You need to update Cruiser.
(current version is 3.0.14)

---

Sie müssen Cruiser aktualisieren.
(die aktuelle Version ist 3.0.14)

**Reply by 1maxi2** [reply]
Wird mir in Play Store nicht angeboten🤔 Habe noch 3.0.12

**Reply by devemux86** [reply]
@1maxi2 

Can you restart your phone and Google Play?

---

Können Sie Ihr Telefon und Google Play neu starten?

**Reply by 1maxi2** [reply]
Herzlichen Dank für deine Geduld !
Nach dem Neustart des Handys wurde mir in Play Store die aktuelle Version vorgeschlagen. Jetzt funktioniert alles 😄

**Reply by ToToGSARider** [reply]
Kann man hier auch Filme hochladen? Ich mache dann morgen mal ein Video was passiert wenn ich am Computer stehe und lade es hier mit der Datei der Route hoch.

Can you also upload movies here? I'll make a video tomorrow times what happens when I stand at the computer and upload it here with the file of the route.

**Reply by MotoUKRider** [reply]
Here's a tip I found with Google Play and Cruiser updates: If you go to "Manage apps and device" from your avatar icon and then select the "Manage" tab - find Cruiser in the list of installed apps and click it.  You should get its homepage with the option to update.

This seems to work ahead of it appearing in the "Overview" tab where your normal pending update list is populated.

I guess you could also search for Cruiser within the Play app and be presented with the same result - I'll try it next time :)

**Reply by devemux86** [reply]
> Can you also upload movies here?

Video uploads are available in GitHub:
- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files

**Reply by 0709wiwiwi** [reply]
> Can you also upload movies here? I'll make a video tomorrow times what happens when I stand at the computer and upload it here with the file of the route.

The most simple is to share and upload these mostly "larger" video files toward and share by YouTube.  
You can directly copy and publish the link here:  https://youtube.com/shorts/S4IOdKvtqPQ
- Eventually even you can add a Title by the link tool in the menu bar here: [My video demo](https://youtube.com/shorts/S4IOdKvtqPQ)
- zip your .kurvig file attachment to publish here: 
[Kurvig.zip](https://github.com/devemux86/cruiser/files/11059479/Kurvig.zip)

**Comment by ToToGSARider** [comment]
@devemux86 wie kann ich denn direkt an dich eine Nachricht schreiben hier in Github? Ich würde Dir dann die Bildschirmaufnahme von meinem Handy senden und die Route. Ich könnte Dir ggf. ja auch im kurviger Forum die Nachricht senden wenn du dort noch angemeldet bist.

@devemux86 how can I write a message directly to you here in Github? I would then send you the screen capture from my phone and the route. I could also send you the message in the kurviger forum if you are still logged in there.

**Reply by devemux86** [reply]
@ToToGSARider 

You can email me at the address shown on the [Cruiser](https://play.google.com/store/apps/details?id=gr.talent.cruiser) page on Google Play.

**Reply by devemux86** [reply]
I created a separate topic to check and discuss the import from the cloud:

- https://github.com/devemux86/cruiser/discussions/96

**Comment by ToToGSARider** [comment]
@devemux86 ich finde das Thema Biker nicht, was mache ich falsch.Wo muss ich das einstellen? Muss ich erst die Offline Karten von Mapsforge laden? 

@devemux86 I can not find the theme Biker, what am I doing wrong.where do I set this? Do I have to load the offline maps from Mapsforge first?

**Reply by 0709wiwiwi** [reply]
https://youtube.com/watch?v=xeyZGTGJcMk&si=EnSIkaIECMiOmarE

**Comment by ToToGSARider** [comment]
@0709wiwiwi das kommt bei mir alles nicht. Ich drücke unten links auf das Symbol mit den 3 Strichen dann kommt bei mir der Block wo ich auf Karte drücken kann und dann kommt bei mir nur Karten herunterladen oder Online Karten. Bei den Online Karten ist Biker aber nicht dabei? Was mache ich falsch?  Muß ich erst Karten herunterladen? Wenn ja welche? 

@0709wiwiwi that does not come with me. I press down left on the icon with the 3 dashes then comes with me the block where I can press on map and then comes with me only download maps or online maps. But Biker is not included in the online maps? What am I doing wrong?  Do I have to download maps first? If so, which ones?

**Reply by devemux86** [reply]
@ToToGSARider

"Biker" map theme is available with the offline maps.

Use the menu "Map | Download maps" to download your preferred offline maps (with the default "Mapsforge" selection).
Downloaded offline maps and "Biker" map theme are automatically activated upon download.

---

Das Kartenthema "Biker" ist bei den Offline-Karten verfügbar.

Verwenden Sie das Menü "Karte | Karten herunterladen", um Ihre bevorzugten Offline-Karten herunterzuladen (mit der Standardauswahl "Mapsforge").
Heruntergeladene Offline-Karten und das Kartenmotiv "Biker" werden beim Herunterladen automatisch aktiviert.

**Comment by ToToGSARider** [comment]
@devemux86 ich bin zu doof. Ich habe die Offline Karte heruntergeladen und das Handy neu gestartet. Bei mir kommen nicht die Auswahlpunkte wie in dem Youtube Video.

@devemux86 I am too stupid. I downloaded the offline map and restarted the phone. I do not get the selection points as in the Youtube video.

**Reply by devemux86** [reply]
@ToToGSARider 

Offline maps are automatically activated when they are downloaded (image 1).
You can also use the 2nd screen to activate the downloaded maps (image 2).

Or can you post an image of the app to see what have you done?

---

Offline-Karten werden automatisch aktiviert, wenn sie heruntergeladen werden (Abbildung 1).
Sie können auch den zweiten Bildschirm verwenden, um die heruntergeladenen Karten zu aktivieren (Bild 2).

Oder können Sie ein Bild der App posten, um zu sehen, was Sie getan haben?

<img src="https://user-images.githubusercontent.com/3484020/227706502-e14bf881-f9b8-4018-bece-17f8a69d423a.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/227706506-e1d6bbd8-46c8-4a35-ab36-7cd89f87f692.jpg" width="33%"/>

**Reply by ToToGSARider** [reply]
Bei mir werden keine Karten angezeigt. Mein Handy sagte Download abgeschlossen? Wenn ich die Karte noch einmal lade, wird dann die vorhandene ggf. überschrieben? Die geladene POIS haben scheinbar auch nicht funktioniert. Oh man, ich möchte einmal erleben, dass auf meinem Samsung S20+ alles so ist wie bei anderen 🤣

No maps are displayed for me. My phone said download complete? If I load the map again, will the existing one be overwritten if necessary? The loaded POIS have apparently not worked either. Oh man, I would like to experience once that on my Samsung S20+ everything is like others. 🤣

**Comment by ToToGSARider** [comment]
Okay, ich darf scheinbar den Downloadbildschirm bzw. die App nicht verlassen für die Zeit des Downloads, dann wird mir die Karte auch angezeigt. Ich probiere es noch einmal.

Okay, I'm apparently not allowed to leave the download screen or app for the time of the download, then the map is displayed to me as well. I'll try it again.

**Reply by devemux86** [reply]
@ToToGSARider 

You can put the app in the background, but you must not close it.
Some manufacturers close the apps in the background, so it is best to wait until the downloads are finished.

---

Sie können die App in den Hintergrund stellen, dürfen sie aber nicht schließen.
Einige Hersteller schließen die Apps im Hintergrund, daher ist es am besten zu warten, bis die Downloads beendet sind.

**Reply by ToToGSARider** [reply]
Das Problem ist die Deutschland Karte. Sie schließt nicht ab.

The problem is the Germany card. It does not lock.
<img src="https://user-images.githubusercontent.com/128730865/227711541-1c7bb247-012d-41fb-9539-8cf18d0e055b.jpg" width="33%"/>

**Reply by ToToGSARider** [reply]
Wenn man auf aktualisieren drückt passiert das.

When you press refresh this is what happens.
<img src="https://user-images.githubusercontent.com/128730865/227711795-dd12acc4-cbeb-46d3-b9f3-f166c06b0093.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/128730865/227711796-46a9d98c-1499-4cb7-b4c8-b65c38bc76d7.jpg" width="33%"/>

**Reply by devemux86** [reply]
It seems this large download does not finish.
Do you keep the app in the foreground or do you close the device screen?
The downloads are done by Android, it shows a notification with a down arrow :arrow_down: so you can check there too.

---

Es scheint, dass dieser große Download nicht beendet wird.
Behalten Sie die Anwendung im Vordergrund oder schließen Sie den Bildschirm des Geräts?
Die Downloads werden von Android durchgeführt, es zeigt eine Benachrichtigung mit einem Pfeil nach unten :arrow_down: so können Sie auch dort überprüfen.

**Reply by devemux86** [reply]
For such large downloads, it is recommended that you keep the app in the foreground
and keep the device screen on until the download is complete.

---

Bei solch umfangreichen Downloads empfiehlt es sich, die App im Vordergrund zu halten
und den Bildschirm des Geräts eingeschaltet zu lassen, bis der Download abgeschlossen ist.

**Reply by WolfgangZ42** [reply]
Same problem here - it took 3 or 4 tries. Finaly it works. 😐

**Reply by ToToGSARider** [reply]
Okay, jetzt hat es funktioniert. Ich habe immer wieder den Bildschirm hell gemacht und bin zwischendurch immer wieder zum Download Manager. Jetzt sieht alles aus wie in meiner K2, ich bin begeistert 🤩
Ich hoffe es ist Okay wenn ich dann das ABO erst am 29.04. für Cruiser mache, denn solange läuft noch das K2 ABO. Sind die Pässe eigentlich in der Karte oder in den POIS? 

Okay, now it worked. I kept brightening the screen and in between I kept going to the download manager. Now everything looks like in my K2, I'm thrilled 🤩
I hope it's okay if I then make the ABO only on 29.04. for Cruiser, because as long as the K2 ABO still runs. Are the passes actually in the map or in the POIS?

**Reply by devemux86** [reply]
Mountain passes are available in:
- Offline maps: with the "Biker" map theme, you enable them in map layers
- Offline poi: all searchable natural OSM data can be seen [here](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-poi-writer/src/main/config/poi-mapping.xml#L894-L911)

---

Bergpässe sind verfügbar in:
- Offline-Karten: mit dem Kartenthema "Biker" aktivieren Sie sie in Kartenebenen
- Offline poi: alle durchsuchbaren natürlichen OSM-Daten sind [hier](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-poi-writer/src/main/config/poi-mapping.xml#L894-L911) zu sehen

**Comment by ToToGSARider** [comment]
Okay, kann ich mal probieren. Ich hatte aber alle Karten markiert und nur die Deutschland Karte ist nicht abgeschlossen worden. Die anderen Karten haben sich ohne Probleme abgeschlossen.

Okay, I can try that. But I had marked all maps and only the Germany map has not been closed. The other maps have closed without problems.

**Reply by devemux86** [reply]
It is also better to not download all maps at the same time, but one after the other.
So the phone's WiFi can download 1 file each time.

---

Es ist auch besser, nicht alle Karten gleichzeitig herunterzuladen, sondern eine nach der anderen.
So kann das WiFi des Telefons jedes Mal 1 Datei herunterladen.

**Reply by MotoUKRider** [reply]
The updates on Mapsforge (I see that it's using the "fast" server) have been hit'n'miss for a couple of days - possibly a result of popular demand?  I had small maps just sitting there not downloading, and it took a few attempts to get the download process to work.

**Reply by ToToGSARider** [reply]
Ach das muss das W-Lan schon können :-) 

Oh the wifi must already be able to do that :-)

**Reply by 0709wiwiwi** [reply]
> It is also better to not download all maps at the same time, but one after the other.
> So the phone's WiFi can download 1 file each time.

Since the download manager is so comfortable, it is also used a lot, so you should check whether there is enough space in the phone or ext sd cards memory. These together  are all quite large downloads. Germany maps + graphs + poi's ......wooow is quite large.
The memory is therefore sometimes faster completely utilised than you think.
128 GB of phone memory has not become an unnecessary luxury.

**Reply by devemux86** [reply]
> you should check whether there is enough space in the phone or ext sd cards memory

There is already a check, it will not start downloading if there is no space available.

**Comment by ToToGSARider** [comment]
<img src="https://user-images.githubusercontent.com/128730865/227716349-8bb598ff-3895-44ea-ac94-d7e805d387ff.jpg" width="33%"/>

So mit welcher APP fahren wir denn heute 🤣

So with which APP we drive today 🤣

**Reply by devemux86** [reply]
All are very nice, I also like OsmAnd very much! :slightly_smiling_face: 

---

Alle sind sehr schön, auch OsmAnd gefällt mir sehr gut! :slightly_smiling_face:

**Reply by 0709wiwiwi** [reply]
OsmAnd. (Super complex).
It's of course like with food colours and smells. Everyone's own taste. 
As for me should I only have OsmAnd I would buy myself a new paper map Atlas right away. 
Ooh well I still kept my older fantastic nice ones ;-) Michelin France etc...
